api/.env.example

```diff
@@ -103,6 +103,10 @@ GCNOTIFY_TEMPLATE_VERIFY_EMAIL_EN=
 GCNOTIFY_TEMPLATE_VERIFY_EMAIL_FR=
 GCNOTIFY_TEMPLATE_SYSTEM_NOTIFICATION_EN=
 GCNOTIFY_TEMPLATE_SYSTEM_NOTIFICATION_FR=
+GCNOTIFY_TEMPLATE_NOMINATION_RECEIVED_SUBMITTER_EN=
```

Comment:
we need to think about some alternative for this .. lol! It keeps accumulating s lot.